test/e2e/pod_create_test.go

```diff
@@ -957,6 +957,34 @@ ENTRYPOINT ["sleep","99999"]
 		Expect(ctr3.OutputToString()).To(ContainSubstring("hello"))
 	})
 
+	It("podman pod create should use customized infra_image", func() {
+		conffile := filepath.Join(podmanTest.TempDir, "container.conf")
+
+		infraImage := "k8s.gcr.io/pause:3.2"
+		err := ioutil.WriteFile(conffile, []byte(fmt.Sprintf("[engine]\ninfra_image=\"%s\"\n", infraImage)), 0644)
+		Expect(err).To(BeNil())
+
+		os.Setenv("CONTAINERS_CONF", conffile)
+		defer os.Unsetenv("CONTAINERS_CONF")
+
+		if IsRemote() {
+			podmanTest.RestartRemoteService()
```

Comment:
Why?  Shouldn't this come from the client?

Reply:
since your PR deals with this better, let's close mine in favor of yours